evap/staff/forms.py

```diff
@@ -48,6 +47,18 @@ class Meta:
         fields = "__all__"
 
 
+class CourseTypeForm(forms.ModelForm, BootstrapMixin):
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+
+        self.fields["name_de"].widget = forms.TextInput(attrs={'class': 'form-control'})
+        self.fields["name_en"].widget = forms.TextInput(attrs={'class': 'form-control'})
+
+    class Meta:
+        model = CourseType
+        fields = "__all__"
+
```

Comment:
i must say i'd feel more comfortable if we checked that for the instance of deleted forms `can_staff_delete` is actually True. in theory, if we added another criterion to `can_staff_delete` that wouldn't be catched by database constraints, we'd likely forget to add that check, to the form deletion.

if i remember correctly, you were able to detect deletion inside `clean()` of this class. what if you raised a SuspiciousOperation or PermissionDenied there instead of ValidationError?

...same would apply for Degree then.